packages/client/components/MeetingCardOptionsMenu.tsx

```diff
@@ -80,16 +91,40 @@ const MeetingCardOptionsMenu = (props: Props) => {
   const {id: viewerId, team, meeting} = viewer
   const {massInvitation} = team!
   const {id: token} = massInvitation
-  const {id: meetingId, meetingType, facilitatorUserId} = meeting!
+  const {id: meetingId, meetingType, facilitatorUserId, meetingSeries} = meeting!
   const isViewerFacilitator = facilitatorUserId === viewerId
   const canEndMeeting = meetingType === 'teamPrompt' || isViewerFacilitator
   const atmosphere = useAtmosphere()
   const {onCompleted, onError} = useMutationProps()
   const {history} = useRouter()
 
+  const hasRecurrenceEnabled = meetingSeries && !meetingSeries.cancelledAt
+
   const {closePortal} = menuProps
   return (
     <Menu ariaLabel={'Edit the meeting'} {...menuProps}>
+      {hasRecurrenceEnabled && (
+        <MenuItem
+          key='link'
+          label={
+            <OptionMenuItem>
+              <LinkIcon />
+              Copy meeting permalink
+            </OptionMenuItem>
+          }
+          onClick={async () => {
```

Comment:
+1 any chance we could use [`CopyLink` component](https://github.com/ParabolInc/parabol/blob/d05e85578a3b6c1b56ebbec1c9bcb37ad326fe1a/packages/client/components/CopyLink.tsx#L14) here to have it all taken care for us?

Reply:
@BartoszJarocki We _could_, but that would be inconsistent with the existing "Copy invite link" logic, and the only immediate benefit would be not needing to call `navigator.clipboard.writeText(copyUrl)` - all other logic is still present, though in a slightly different place.